tests/fetchai/test_communication.py

```diff
@@ -0,0 +1,74 @@
+import json
+import uuid
+
+import pytest
+from requests_mock import Mocker as RequestsMocker
+
+from fetchai import communication
+from fetchai.crypto import Identity
+
+
+@pytest.fixture
+def identity() -> Identity:
+    return Identity.from_seed("TESTING", 1)
+
+
+def fake_agent_lookup_function(agent_address: str) -> str:
+    """A fake agent look up function to use in test.
+
+    Returns a predictable URL for testing against"""
+
+    return f"http://localhost/fake_endpoint/{agent_address}"
+
+
+class TestSendMessageToAgent:
+    def test_happy_path(self, identity: Identity):
+        agent_name = "agent_one"
+        expected_url = "http://localhost/fake_endpoint/agent_one"
```

Comment:
```suggestion
        expected_url = fake_agent_lookup_function(agent_name)
```
Either this, or creating a template that both the lookup_function and the expected_url call into, so that the coupling between these strings is less brittle?